test-lit/lower/intrinsic-operations.f90

```diff
@@ -0,0 +1,12375 @@
+! Generated test file
```

Comment:
Do we want to checkin the generated output here or have a test the generates the output as part of the test?

Reply:
To the latter point: one can build up a "shell script" with multiple RUN lines.

Reply:
Agreed this should not be checked in, it was just for review purpose. I liked the idea of having the file generated somewhere though for easier failure debug, but the "shell script" option might also be good with `–dump-input=fail`.